geomesa-kafka/geomesa-kafka-08-datastore/src/main/scala/org/locationtech/geomesa/kafka08/KafkaProducerFeatureStore.scala

```diff
@@ -139,13 +139,13 @@ class KafkaFeatureWriterModify(sft: SimpleFeatureType, producer: Producer[Array[
 
 object KafkaProducerFeatureStoreFactory {
 
-  private [kafka] val FeatureIds = new AtomicLong(0L)
+  private [kafka08] val FeatureIds = new AtomicLong(0L)
 
   def apply(broker: String): FeatureSourceFactory = {
 
     val config = {
       val props = new ju.Properties()
-      props.put("metadata.broker.list", broker)
+      props.put(new KafkaUtils08().brokerParam(), broker)
```

Comment:
(picky) Also, if ``brokerParam`` is only returning the name of the parameter without a side effect, the Scala convention is to omit the empty parens.